github.py

```diff
@@ -111,15 +111,19 @@ class ImageTableDirective(Table):
     }
 
     def run(self):
-        cols = self.options.get("columns", 3)
+        cols = self.options.get("columns", 4)
```

Comment:
IMO 3 is already pushing it for mobile, until there can be a different amount of columns for desktop/mobile I don't think this should change.

Reply:
```suggestion
        cols = self.options.get("columns", 3)
```